cmd/zetaclientd/start.go

```diff
@@ -309,3 +316,26 @@ func initPreParams(path string) {
 		}
 	}
 }
+
+// promptPasswords() This function will prompt for passwords which will be used to decrypt two key files:
+// 1. HotKey
+// 2. TSS key-share
+func promptPasswords() (string, string, error) {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Print("HotKey Password: ")
+	hotKeyPass, err := reader.ReadString('\n')
+	if err != nil {
+		return "", "", err
+	}
+	fmt.Print("TSS Password: ")
+	TSSKeyPass, err := reader.ReadString('\n')
+	if err != nil {
+		return "", "", err
+	}
+
+	if TSSKeyPass == "" || hotKeyPass == "" {
+		return "", "", errors.New("hotkey and tss passwords are required to start zetaclient")
```

Comment:
Does it mean providing a password is required and that keyring backend test is no longer supported?

Reply:
I will have to test this. If we want to allow "test" backend still then I'll make sure it still works.

Reply:
I think it would be better, yes.

Reply:
works now. Also ported it to develop: https://github.com/zeta-chain/node/pull/1630